doc/classes/BitMap.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		A two-dimensional array of boolean values, can be used to efficiently store a binary matrix (every matrix element takes only one bit) and query the values using natural cartesian coordinates.
+		[b]Note:[/b] The maximum size of a [BitMap] is [code]2^31 - 8[/code] values (inclusive). For a square bitmap, the maximum size is therefore [code]46340×46340[/code]. For a 1 value-wide or tall bitmap, the maximum size is [code]2147483640×1[/code] or [code]1×2147483640[/code]. Writing the [BitMap] to an image has stricter size restrictions; see [method convert_to_image].
```

Comment:
```suggestion
		[b]Note:[/b] The maximum size of a [BitMap] is [code]2^31 - 8[/code] pixels (inclusive). For a square bitmap, the maximum size is therefore [code]46340×46340[/code]. For a 1 pixel-wide or tall bitmap, the maximum size is [code]2147483640×1[/code] or [code]1×2147483640[/code]. Writing the [BitMap] to an image has stricter size restrictions; see [method convert_to_image].
```
Or use `values` in the errors, IMO

Reply:
I've changed it to use "values" in the errors, as BitMap is not an image until you convert it to an Image (it doesn't have "pixels" until then).